### ISSUE TITLE
Accessibility: stop VoiceOver from reading out text behind hint alerts

### ISSUE THREAD

**Issue by twostraws** [twostraws/Unwrap#225]
From an app user: "The pop up modal hints, that sometimes appear in the app, still allow a VoiceOver user to access the content behind the modal in the viewport. This makes it very difficult to know what content is actually inside the modal, and can be disorienting."

Solution: we need to make sure when our alerts appear VoiceOver doesn't read anything behind. If this isn't possible with SwiftEntryKit then perhaps we should file a request there, because it seems like a sensible behavior.

**Comment by gvmrtnv** [comment]
Could you please give an example of these "modal hints"? 
I've tried to reproduce this false behaviour with Hint alerts in Practice Tab, but VoiceOver doesn't read anything behind.  

![image](https://user-images.githubusercontent.com/94224033/147383337-df1a853f-3731-4945-be0b-1688449534d0.png)

**Comment by EthanChinCN** [comment]
This behavior has been implemented by SwiftEntryKit.
See in this PR:[feature(Accessibility): add accessibility capabilities to button. make every EKWindow instance a modal for accessibility](https://github.com/huri000/SwiftEntryKit/pull/304)
Update SwiftEntryKit to version 1.2.5 or above to fix this issue.

**Comment by swif2** [comment]
Seems this has been fixed.
